bentoml/models.py

```diff
@@ -204,20 +205,31 @@ def export_model(
     )
 
 
+@inject
 def push(
     tag: t.Union[Tag, str],
     *,
+    force: bool = False,
     _model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
 ):
-    raise NotImplementedError
+    from bentoml._internal.yatai_client import yatai_client
+
+    model_obj = _model_store.get(tag)
+    if not model_obj:
+        raise BentoMLException(f"Model {tag} not found in local store")
```

Comment:
```suggestion
        raise BentoMLException(f"Model {tag} not found in local store.")
```

Reply:
let's just keep it consistent with the yatai CLI command implementation for now, this is just copied over that code. It will need a bit of refactoring all together, will do in follow up

Reply:
sg